modules/lvms-creating-lvms-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-lvms-cluster-using-web-console_{context}"]
+= Creating an LVMCluster CR by using the web console
+
+You can create an `LVMCluster` CR on a worker node using the {product-title} web console.
+
+[IMPORTANT]
+====
+You can only create a single instance of the `LVMCluster` custom resource (CR) on an {product-title} cluster.
+====
+
+.Prerequisites
+
+* You have access to the {product-title} cluster with `cluster-admin` privileges.
+
+* You have installed {lvms}.
+
+* You have installed a worker node in the cluster.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Click *Operators* -> *Installed Operators*.
+. In the `openshift-storage` namespace, click *LVM Storage*.
+. Click *Create LVMCluster* and select either *Form view* or *YAML view*.
+. Configure the required `LVMCluster` CR parameters.
+. Click *Create*.
+. Optional: If you want to edit the `LVMCLuster` CR, perform the following actions:
+.. Click the *LVMCluster* tab.
+.. From the *Actions* menu, select *Edit LVMCluster*. 
+.. Click *YAML* and edit the required `LVMCLuster` CR parameters.  
+.. Click *Save*.
+
+.Verification
+
+. On the *LVMCLuster* page, check that the `LVMCluster` CR is in the `Ready` state. 
```

Comment:
This still needs to be changed to unnumbered

```suggestion
* On the *LVMCLuster* page, check that the `LVMCluster` CR is in the `Ready` state. 
```

Reply:
I am sorry. I missed to notice this.
Updated it to unnumbered list.